docs/content/docs/tasks_usage/metrics.md

```diff
@@ -29,7 +29,7 @@ to automatically provision the metrics components and with the BYO method.
 
 By default, both Prometheus and Grafana are disabled.
 
-However, when configured with the `--deploy-grafana` flag, OSM installation will deploy a Prometheus instance to scrape the sidecar's metrics endpoints. Based on the metrics scraping configuration set by the user, OSM will annotate pods part of the mesh with necessary metrics annotations to have Prometheus reach and scrape the pods to collect relevant metrics. To install Grafana for metrics visualization, set the `deploy-grafana` flag to true when installing OSM using the `osm install` command.
+However, when configured with the `--deploy-prometheus` flag, OSM installation will deploy a Prometheus instance to scrape the sidecar's metrics endpoints. Based on the metrics scraping configuration set by the user, OSM will annotate pods part of the mesh with necessary metrics annotations to have Prometheus reach and scrape the pods to collect relevant metrics. To install Grafana for metrics visualization, set the `--deploy-grafana` flag to true when installing OSM using the `osm install` command.
```

Comment:
Would it make sense for this doc to be under `tasks_usage/observability`?

Reply:
Makes sense to me. I think I even remember it being there at one point. I'd prefer to move it in a separate PR though to handle updating links both to and from this doc.